sources/test.py

```diff
@@ -0,0 +1,32 @@
+import random
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+from Tools.LogReg import LogReg
+from Tools.ArgParser import ArgParser
+from Tools.DatasetHandler import DatasetHandler
+
+scores = []
+
+
+def test():
+    args = ArgParser()
+    dataset = DatasetHandler(args, parse=True, train=True)

```

Comment:
# check if args of DatasetHandler is correct
See in the code or on Discord, i've add the same comment.